### ISSUE TITLE
SQL from Expression param is empty after first call sql() method

### ISSUE THREAD

**Issue by kfreiman** [shadowhand/latitude#6]
Version = 1.1.0

### Code to reproduce
```php
use Latitude\QueryBuilder\InsertQuery;
use Latitude\QueryBuilder\Expression;

$query = InsertQuery::make('t1', [
    'created_at' => Expression::make('NOW()')
]);

var_dump($query->sql());  // INSERT INTO t1 (created_at) VALUES (NOW())
var_dump($query->sql());  // INSERT INTO t1 (created_at) VALUES ()
```
### Expected
Each `$query->sql()` should return the same.

**Comment by shadowhand** [comment]
Doh, of course. Not sure how I missed this when I designed that feature.

**Comment by shadowhand** [comment]
Fixed in version 1.1.1
